src/pages/operations/actions/RefreshOperationsBtn.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, useEffect } from "react";
+import { useQueryClient } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { ActionButton, Icon } from "@canonical/react-components";
+import { useOperations } from "context/operationsProvider";
+
+const RefreshOperationsBtn: FC = () => {
+  const { isFetching } = useOperations();
+  const queryClient = useQueryClient();
+
+  const handleRefresh = () => {
+    void queryClient.invalidateQueries({ queryKey: [queryKeys.operations] });
+  };
+
+  // force a refresh on first render
+  useEffect(handleRefresh, []);
+
+  const handleBtnClick = () => {
```

Comment:
I think this handler is redundant, we could move the logic from `handleRefresh` into `handleBtnClick`.

Reply:
This is a leftover, before the fetching/loading state was locally in this component and resulted in a need for a dedicated function to issue the loading state. Now it can totally be removed. I'll fix it shortly.